<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentWidget</class>
 <widget class="QWidget" name="CommentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>标注</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource/image.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(60, 60, 60);
	color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <widget class="QFrame" name="frame_Btn">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_Note">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}
QPushButton:disabled {
    background-color: rgb(85, 85, 85);
}</string>
        </property>
        <property name="text">
         <string>添加标记</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Last">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}
QPushButton:disabled {
    background-color: rgb(85, 85, 85);
}</string>
        </property>
        <property name="text">
         <string>上一条</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Next">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}
QPushButton:disabled {
    background-color: rgb(85, 85, 85);
}</string>
        </property>
        <property name="text">
         <string>下一条</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame_CMT">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_CMT">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>评论：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextBrowser" name="textBrowser_Comment">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(80, 80, 80);
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_AppName">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_AppName">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>应用名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_AppNameDisplay">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar{
	background: rgb(200, 200, 200);
	text-align: right;
	color: rgb(255,255,250);
	border-radius:5px;
}
QProgressBar::chunk{
	border-radius:2px;
	background-color:rgb(80, 80, 80);
	width:8px;
	margin: 0.5px;
}</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="format">
         <string>%v/%m</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_CMTTheme">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_CMTTheme">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>主题：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_CMTThemeDisplay">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_AppValue">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_AppValue">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>评分：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_AppValueDiaplay">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="5">
    <widget class="QToolButton" name="toolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QToolButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QToolButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource/image.qrc"/>
 </resources>
 <connections/>
</ui>
